index.js

```diff
@@ -22,7 +22,7 @@ let { connectToMysql } = require("./db");
 const app = express();
 
 const corsOptions = {
-  origin: ["http://localhost:5173"], //this will allow multiple domains to connect
+  origin: ["http://localhost:5174"], //this will allow multiple domains to connect
```

Comment:
why changed this configuration. this will blow down everybody work, comma lagakr apna url nahi daal sakte?

Reply:
done by adnan